src/main/java/codeu/model/data/User.java

```diff
@@ -18,11 +18,12 @@
 import java.util.UUID;
 
 /** Class representing a registered user. */
-public class User {
-  private final UUID id;
+public class User extends Activity{
```

Comment:
A `User` extending `Activity` doesn't make sense. You seem to be using `User` for 'login' as an activity. If yes, it might make more sense to have that as a separate class (or do it in a future PR, skip login tracking for now).